dd-sdk-android/src/main/kotlin/com/datadog/android/rum/internal/domain/scope/RumViewScope.kt

```diff
@@ -742,7 +751,8 @@ internal open class RumViewScope(
                     session = ViewEvent.ViewEventSession(
                         id = rumContext.sessionId,
                         type = ViewEvent.ViewEventSessionType.USER,
-                        hasReplay = hasReplay
+                        hasReplay = hasReplay,
+                        isActive = rumContext.isSessionActive
```

Comment:
I believe we are hitting the problem here by relying on the `rumContext`. Imagine the following situation: session was stopped, but view needs to wait for the child events to complete. But before they complete, another session was started, meaning by the time this view will receive terminal view update, it will be another session in the `RumContext` (`active=true` with another session ID), different from the one which view was attributed to originally.

I think the only solution in this case is to take session ID at the time view is constructed and rely on it, instead of relying on the shared `RumContext`.